english/coffee/novel/full-length-novel.md

```diff
@@ -1,3 +1,11 @@
 Driven by the massive dose of caffeine, you write a full-length novel
 about the existential angst of post-modernism and the effects it has on
-the average citizen of the global industrial complex.
\ No newline at end of file
+the average citizen of the global industrial complex.
+
+You just written a full-length novel, now what do you do?
+
+[Sell the movie rights](../coffee/hollywood.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.